Modelica_LinearSystems2/WorkInProgress/Controller/KalmanFilter/UKF_SR.mo

```diff
@@ -1,6 +1,7 @@
 within Modelica_LinearSystems2.WorkInProgress.Controller.KalmanFilter;
 model UKF_SR "Unscented Kalman filter"
   import Modelica;
+  import MSLcholesky = Modelica.Math.Matrices.cholesky;

```

Comment:
Same: importing a function with a different name looks weird. I would either import it as just "cholesky" or import MatricesMSL as in other places and use MatricesMSL.cholesky

Reply:
Done.